pkg/resources/machinecontroller/deployment.go

```diff
@@ -53,7 +53,7 @@ var (
 const (
 	Name = "machine-controller"
 	// TODO: update to tagged machine-controller version.
-	Tag = "0a94443b41516398b5bd6a129589dea599f3e7b8"
+	Tag = "e7e4ba8eb535086374108ed235c2e33bab7f4292"
```

Comment:
@hdurand0710 do you want to keep that commit hash?

Reply:
nope, that would need to be the one after you merge https://github.com/kubermatic/machine-controller/pull/1529